<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RELAY_TEST</class>
 <widget class="QMainWindow" name="RELAY_TEST">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="relay1_on">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Relay-1 ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="relay2_on">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Relay-2  ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="relay1_off">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Relay-1 OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="relay2_off">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Relay-2 OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_relay_window">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="relay_status">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
